### PR TITLE
Migrate to D88 wfs and clean up

### PR DESCRIPTION
**Pull request by srimanob**
#### PR description:
This PR follows the green light of D88 validation report at PPD General ([March 17](https://indico.cern.ch/event/1139390/))

The PU workflow of D88 is added to IB. In addition, DD4hep wf of D88 is added to the short matrix, and there is a clean up in the list of the short matrix and IB workflows.

#### PR validation:
-

#### if this PR is a backport please specify the original PR and why you need to backport that PR:
Not a backport, and no need of backport.

### CODE REVIEW

**Comment by srimanob** [comment]
@cmsbuild please test

**Comment by srimanob** [comment]
+Upgrade

This PR moves the Phase-2 baseline to D88, and do clean up the obsolete workflows out from short matrix and IB.

**Comment by srimanob** [comment]
test parameters:
- workflows = 39461.97,39434.9,39434.103

**Comment by srimanob** [comment]
@cmsbuild please test

**Comment by srimanob** [comment]
test parameters:
- workflows = 39661.97,39434.9,39434.103

**Comment by srimanob** [comment]
@cmsbuild please test

**Comment by kskovpen** [comment]
+pdmv

**Comment by perrotta** [comment]
+1

**Comment by srimanob** [comment]
There is an issue with step-2 in IB. I assume it relates to the MinBias. However, the PR is tested also with MinBias, i.e. 39661.97, not sure why it is the case.

```
----- Begin Fatal Exception 19-Mar-2022 01:22:39 CET-----------------------
An exception of category 'NoSecondaryFiles' occurred while
   [0] Constructing the EventProcessor
   [1] Constructing module: class=MixingModule label='mix'
Exception Message:
RootEmbeddedFileSequence no input files specified for secondary input source.
----- End Fatal Exception -------------------------------------------------
```

**Comment by srimanob** [comment]
There is an issue with step-2 in IB. I assume it relates to the MinBias. From the IB log, it tries to pick up the MinBias dataset as follows, which exists if I check with DAS. Any idea on what exactly happen @qliphy @perrotta @smuzaffar ? Thanks.

IB log:
```
the query is file dataset = /RelValMinBias_14TeV/CMSSW_12_3_0_pre5-123X_mcRun4_realistic_v4_2026D88noPU-v1/GEN-SIM
2022/03/19 01:21:59 ERROR failed to parse X509 proxy: crypto/tls: failed to parse key: asn1: syntax error: sequence truncated
```

Error:
```
----- Begin Fatal Exception 19-Mar-2022 01:22:39 CET-----------------------
An exception of category 'NoSecondaryFiles' occurred while
   [0] Constructing the EventProcessor
   [1] Constructing module: class=MixingModule label='mix'
Exception Message:
RootEmbeddedFileSequence no input files specified for secondary input source.
----- End Fatal Exception -------------------------------------------------
```

**Comment by qliphy** [comment]
@srimanob   There is also a related unit failure in the same IB:
https://cmssdt.cern.ch/SDT/cgi-bin/logreader/slc7_amd64_gcc10/CMSSW_12_4_X_2022-03-18-2300/unitTestLogs/Configuration/PyReleaseValidation#/
2022/03/19 00:12:46 ERROR failed to parse X509 proxy: crypto/tls: failed to parse key: asn1: syntax error: data truncated

@smuzaffar may comment more, but could it be a glitch?

Anyway I tried to run 39634.21 locally with this IB  and everything works well...

**Comment by smuzaffar** [comment]
This is a new das query which was never cached by our system. So during relval tests we used direct `dasgoclient` which failed ( for some unknown reason). I have also seen this das failure [here](https://cmssdt.cern.ch/SDT/cgi-bin/logreader/slc7_amd64_gcc10/CMSSW_12_4_X_2022-03-18-2300/unitTestLogs/PhysicsTools/Utilities#/1569-1569).
Anyway, this query has been cached hours ago  [a] , so this workflow should run fine for 11h00 IB.

[a]
https://github.com/cms-sw/cms-sw.github.io/blob/master/das_queries/72/729f9bbd0b21b91e52006b48fa7f00dc515a947e7fa53b6e88b36bbe1ae7707b.query
https://github.com/cms-sw/cms-sw.github.io/blob/master/das_queries/72/729f9bbd0b21b91e52006b48fa7f00dc515a947e7fa53b6e88b36bbe1ae7707b

File: Configuration/PyReleaseValidation/python/relval_2026.py [hunk 1]
```diff
@@ -15,20 +15,13 @@
 #2026 WFs to run in IB (TTbar)
 numWFIB = []
 numWFIB.extend([23234.0]) #2026D49
-numWFIB.extend([23461.97]) #2026D49 premixing stage1 (NuGun+PU)
 numWFIB.extend([23434.99,23434.999]) #2026D49 premixing combined stage1+stage2 (ttbar+PU200, ttbar+PU50 for PR test)
 numWFIB.extend([23234.21,23434.21,23434.9921]) #2026D49 prodlike, prodlike PU, prodlike premix stage1+stage2
-numWFIB.extend([23234.103]) #2026D49 aging
-numWFIB.extend([23234.9]) #2026D49 vector hits
```
[the remark this answers is not in the document]
Comment (srimanob) [reply Configuration/PyReleaseValidation/python/relval_2026.py:22]: I moved it to D88, `39434.9`. Do you still need it for D49, for PR test?

[the remark this answers is not in the document]
Comment (mmusich) [reply Configuration/PyReleaseValidation/python/relval_2026.py:22]: Hi @srimanob, no that's fine, provided there is at least one VH workflow exercised in IBs, I didn't see it was migrated, sorry.

**Pull request merged**